src/radix_connect/wallet_account.rs

```diff
@@ -4,33 +4,37 @@ use crate::prelude::*;
 #[serde(rename_all = "camelCase")]
 pub struct WalletInteractionWalletAccount {
```

Comment:
Oh ok, we have a duplicated model, this and the model type in `AuthorizedPersonaDetailed` SHOULD be the SAME , and `AuthorizedPersonaDetailed` is using `AccountForDisplay`. 

So we can change `AuthorizedPersonaDetailed` to use this model type. But you would need to make a few changes for that.

Then ensure `AccountsForDisplay` is only used by `sargon_os_accounts` ([see this](https://github.com/radixdlt/sargon/blob/ac/system_with_drivers/src/system/sargon_os/sargon_os_accounts.rs#L19)) , and then we are free to modify `AccountsForDisplay` to what ever format we want (and it need not be Serde).

Reply:
I can create a separate PR for this change as it is not directly related to the current one.